contracts/scripts/native_solc_compile_all_ocr2vrf

```diff
@@ -7,10 +7,13 @@ echo $SCRIPTPATH
 python3 -m pip install --require-hashes -r $SCRIPTPATH/requirements.txt
 solc-select install 0.8.15
 
+## Change me.
```

Comment:
Keeping this comment?

Reply:
Yes! For use by us.

Reply:
We change to the correct directory when actually running the script.